api/serializers.py

```diff
@@ -3,14 +3,16 @@
 from .models import User, GameSession, Court, CourtAddress, UserAddress, Guest, Profile, AddressModelMixin
 
 class UserSerializer(serializers.ModelSerializer):
-    
+    # profile = ProfileSerializer(source='profile', read_only=True)
+
     class Meta:
         model = User
         fields = [
             'id',
             'username',
             'first_name',
             'last_name',
+            'profile',
```

Comment:
Maybe we should remove this line from the generic user serializer and only apply to the user detail serialzer

Reply:
Yep, good catch.